package.json

```diff
@@ -112,6 +112,7 @@
     "lint-staged": "15.2.2",
     "mocha": "10.3.0",
     "prettier": "3.2.5",
+    "ts-retry": "^4.2.5",
```

Comment:
Can we just use `while (retry--) { try { await test() } catch(e) { /**/} }`?
The ts-retry looks like an overkill here.

Reply:
Good suggestion, this winds up being much simpler!